files/galaxy/subdomains/live/themes/live.yml

```
@@ -0,0 +1,9 @@
+live:
+  masthead:
+    color: >
+      linear-gradient(33deg, rgba(131,58,180,1) 0%,
+      rgba(253,29,29,1) 25%, rgba(252,176,69,1) 50%,
+      rgba(253,29,29,1) 75%, rgba(131,58,180,1) 100%),
+      linear-gradient(33deg, rgba(131,58,180,1) 0%,
+      rgba(253,29,29,1) 25%, rgba(252,176,69,1) 50%,
+      rgba(253,29,29,1) 75%, rgba(131,58,180,1) 100%)
```

```suggestion
      linear-gradient(33deg, rgba(131,58,180,1) 0%,
          rgba(253,29,29,1) 25%, rgba(252,176,69,1) 50%,
          rgba(253,29,29,1) 75%, rgba(131,58,180,1) 100%)
```

since these overlap and are identical, one gradient should render the same result